docs/source/privacy_policies.rst

```diff
@@ -67,13 +67,18 @@ Users can also disable the plugin by executing:
 External web services
 ---------------------
 
-GitHub Pages
-^^^^^^^^^^^^
+JupyterLab Docs site
+^^^^^^^^^^^^^^^^^^^^
 
-The service hosting the https://jupyterlab.github.io website stores access logs.
-That data is not accessible to and not readable by the JupyterLab development team.
```

Comment:
We do still make programatic calls to jupyterlab.github.io/assets (as in https://github.com/jupyterlab/assets/pull/4) and that is from within the app itself (well, docs pages can be accessed from within the app too via the Help menu, so both belong here for comprehensiveness). I think the opt-in notification even links to this privacy policy. New versions get checked from GitHub too, not sure which exact domain.

Reply:
Thanks Mike. I just put back the github pages policy, instead making a new section for the plausible privacy policy.